main.go

```diff
@@ -10,6 +10,8 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var version = "0.0.0"
```

Comment:
hehe yea it was tricky b/c the actual `LDFLAGS` line was already in code on master...